Makefile

```diff
@@ -241,6 +241,10 @@ deploy: manifests kustomize ## Deploy controller to the K8s cluster specified in
 undeploy: kustomize ## Undeploy controller from the K8s cluster specified in ~/.kube/config. Call with ignore-not-found=true to ignore resource not found errors during deletion.
 	$(KUSTOMIZE) build config/default | $(KUBECTL) delete --ignore-not-found=$(ignore-not-found) -f -
 
+.PHONY: release
```

Comment:
`release` itself is a little bit confusing. Please follow similar pattern like `install-xx` or `deploy-xxx`

Reply:
rename to deploy-release.